src/main/resources/merge-patient.ktr

```diff
@@ -0,0 +1,1119 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please provide a folder structure here for the PDI transformations.

Something like: jobs, transformations and settings

Reply:
Sure. This files end-up inside the `jar` after the build, does it make sense to have settings in there?

Reply:
So I'd like us to include at least a sample settings file that the partners can just update. We can maybe include that under ticket MEPTS-85